vendor/github.com/docker/swarmkit/vendor.conf

```diff
@@ -28,18 +28,16 @@ github.com/docker/libnetwork 37e20af882e13dd01ade3658b7aabdae3412118b
 github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 github.com/opencontainers/runc b6b70e53451794e8333e9b602cc096b47a20bd0f
 github.com/opencontainers/go-digest a6d0ee40d4207ea02364bd3b9e8e77b9159ba1eb
-github.com/opencontainers/image-spec f03dbe35d449c54915d235f1a3cf8f585a24babe
+github.com/opencontainers/image-spec 372ad780f63454fbbbbcc7cf80e5b90245c13e13
```

Comment:
Agreed - just wanted to point out that we shouldn't be concerned about swarmkit being tested against a different version of this dependency, since it's not a factor for the Docker executor. If we bump it, it should be because we want to keep up-to-date, not because it's important to match swarmkit's version.